nav2_bringup/launch/navigation_launch.py

```diff
@@ -155,6 +156,15 @@ def generate_launch_description():
                 respawn_delay=2.0,
                 parameters=[configured_params],
                 remappings=remappings),
+            Node(
```

Comment:
even if this is in the default launch file, by default, the simulated turtlebot won't be affected by this right since the smoother publishes on the smoothed topic?

Reply:
Correct. A similar thing could be said about the waypoint follower and path smoother server

Reply:
To enable, you should remap the `cmd_vel` topic of  the controller server to `cmd_vel_nav` (or similar) and then remap the smoother to use that as the input. Then probably remap `cmd_vel_smoothed` to `cmd_vel` so it passes through this. I'll add to my TODO list to add a tutorial for how to do this (or just have it be the default bringup)